uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/api/EventStorage.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.mqtt.client.control.api;
+
+import lombok.NonNull;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Interface to storage of the events.
+ */
+public interface EventStorage {
+
+    /**
+     * Removes all events from storage.
+     */
+    void clear();
+
+    /**
+     * Adds event to storage.
+     *
+     * @param event the event to add
+     */
+    void addEvent(@NonNull Event event);
+
+    /**
+     * Gets matched events from storage immediately.
+     *
+     * @param filter the filter of events
+     * @return the array of matched events
+     */
+    Event[] getEvents(@NonNull EventFilter filter);
```

Comment:
I'd like to change arrays to something from Collections API. It is a common practice to use List - https://docs.oracle.com/javase/8/docs/api/java/util/List.html

Reply:
Fixed in a5267e0af6fd266ee4f0465106cf4f315bfbf526